native/cocos/renderer/pipeline/custom/RenderGraphTypes.h

```diff
@@ -192,12 +192,17 @@ struct RenderSwapchain {
     : swapchain(swapchainIn),
       isDepthStencil(isDepthStencilIn) {}
 
+    static gfx::Texture* getColorTexture(gfx::Swapchain* swapchain) noexcept;
+    static gfx::Texture* getColorTexture(scene::RenderWindow* renderWindow) noexcept;
+    static gfx::Texture* getDepthStencilTexture(gfx::Swapchain* swapchain) noexcept;
```

Comment:
What are these functions invoked?

Reply:
They are invoked when registering `RenderWindow` to the pipeline.